code/game/verbs/ooc.dm

```diff
@@ -138,3 +138,23 @@ var/global/normal_ooc_colour = "#002eb8"
 		return
 
 	show_note(usr, null, 1)
+
+/client/proc/ignore_key(client)
+	var/client/C = client
+	prefs.ignoring ^= C.key

```

Comment:
```^=``` doesn't seem like the right operator here.
1, C.key isn't a list or a number, so this isn't the list operation or the XOR operation
2. If it was a list you'd be setting A equal to just the unique elements of B, which really makes no sense as just a ```|=``` and a ```-=``` in an if/else would do it.

you're